swal-forms.js

```diff
@@ -46,10 +46,13 @@
     addWayToGetFormValuesInDoneFunction: function(swalArgs) {
       var swalFormInstance = this
       var doneFunction = swalArgs[1]
-      swalArgs[1] = function() {
+      swalArgs[1] = function(isConfirm) {
```

Comment:
I see the same modal and behaviors with the 2 swal calls described in the comment above. On both cases a swal modal is displayed with just an "OK" button which when clicked opens an alert. Do you observe a different behavior?